external/dhcpc/dhcpc.c

```diff
@@ -178,6 +178,9 @@ struct dhcpc_state_s {
 	struct dhcp_msg packet;
 };
 
+
+int total_dns;
```

Comment:
If this variable is used only in this file, please add static.

Reply:
Done